create-vm.sh

```diff
@@ -0,0 +1,14 @@
+if (multipass info awesome-vm)
+ then
+  echo "The VM already exists"
+ else
+  echo "✨ Creating an awesome vm for you. One moment... ✨"
+  multipass launch --name awesome-vm
+fi
+
+
+if (multipass info awesome-vm)
```

Comment:
You don't need a conditional here and I would move the shell command outside of the script to the README.

Reply:
I took your suggestion and removed the conditional